.gitignore

```diff
@@ -21,3 +21,4 @@ apk-sizes-*.txt
 *.binlog
 src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props
 *~
+external/monodroid/
```

Comment:
Should just all of `external` be ignored?
```suggestion
external/
```

If we add new ones, this wouldn't have to change.

Reply:
We're using that as the submodule root as well, so I don't think we want to ignore everything in there. That would mean that any submodule bumps would also be ignored.

Reply:
🤦‍♂️ duh